run_unsupport.go

```diff
@@ -1,11 +1,32 @@
-// +build !linux
+// +build windows
```

Comment:
Call this run_windows.go

Reply:
If you do, does @baude need to create a separate run_unsupported with empty functions?